cmd/protoc-gen-crd/openapiGenerator.go

```diff
@@ -697,6 +711,14 @@ func (g *openapiGenerator) generateDescription(desc protomodel.CoreDesc) string
 }
 
 func (g *openapiGenerator) fieldType(field *protomodel.FieldDescriptor) *apiext.JSONSchemaProps {
+	if !g.includeExperimentalFields {
+		if gt := parseMessageGenTags(field.Location().GetLeadingComments()); gt != nil {
+			if gt["releaseChannel"] == "standard" {
```

Comment:
Should this conditional also be checking `stable`?

Reply:
Same as above, Only standard  fields will have a `releaseChannel` tag. So this is checking to see if the resource has a tag `releaseChannel:standard`, then if `includeExperimental` is false skip the generation of the standard field in the stable channel.